beacon-chain/blockchain/service.go

```diff
@@ -83,6 +90,7 @@ func NewService(ctx context.Context, cfg *Config) (*Service, error) {
 		forkChoiceStore:      store,
 		chainStartChan:       make(chan time.Time),
 		stateInitializedFeed: new(event.Feed),
+		headUpdatedFeed:      new(event.Feed),
```

Comment:
but where is this new feed used ? I just see it being defined now. Or is that for another PR. if it is can you add a TODO for it